app/src/androidTest/java/org/systers/mentorship/view/fragments/ChangePasswordFragmentTest.kt

```diff
@@ -0,0 +1,140 @@
+package org.systers.mentorship.view.fragments
+
+import android.view.View
+import android.widget.EditText
+import androidx.annotation.IdRes
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.ViewInteraction
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.RootMatchers.isDialog
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import androidx.test.rule.ActivityTestRule
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.CoreMatchers
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.systers.mentorship.R
+import org.systers.mentorship.view.activities.SettingsActivity
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+class ChangePasswordFragmentTest {
+
+    private val PASSWORD_TOO_WEAK = "Your password is too weak! Use at least one small and capital letter, one number and one special sign!"
```

Comment:
is there any way to get this from `strings.xml` @justdvnsh ? Out of curiosity, I am not experienced in writing tests for android

Reply:
can be done. Should I ?

Reply:
My worry, is if the message changes on strings.xml then this would have to be updated, if we could do something like `getString(R.string....)` that would be good right @justdvnsh ?

Reply:
yup

Reply:
@isabelcosta DONE. :) 

![test_resolved_issue](https://user-images.githubusercontent.com/29144058/97294176-a8f70a00-1873-11eb-88f8-48b54e00de3b.png)